bootstrap.py

```diff
@@ -112,7 +112,7 @@ def link_system_libs():
                 'krbVmodule', 'deltarpm', '_deltarpmmodule',
                 'fedora_cert', 'libxml2', 'libxml2mod', 'librepo', 'createrepo_c',
                 'dnf', 'libcomps', 'gpgme', 'lzma', 'iniparse', 'hawkey',
-                'yum'):
+                'yum', 'pungi'):
```

Comment:
The Pungi dependency should also be added to ```requirements.txt```, ```devel/ansible/roles/dev/tasks/main.yml```, and mentioned in ```docs/release_notes.rst```.

Reply:
All of the above were added except the release notes - I'll send you a PR to add that.

Reply:
I don't know what version of ```pungi``` is required. @lubomir, should I just say "pungi >= 4.0.0"?

Reply:
I'm going to say 4.1.18, since that's the version that we've tested on F25, unless someone speaks up and says we can do an older version than that.

Reply:
I will also add the version to the requirements.txt file.

Reply:
https://github.com/mcurlej/bodhi/pull/23